builtins/src/test/java/org/jline/builtins/PosixCommandsTest.java

```diff
@@ -119,6 +119,8 @@ void testLsBasic() throws Exception {
         PosixCommands.ls(context, new String[] {"ls"});
 
         String output = out.toString();
+        assertTrue(output.contains(".\n"));
```

Comment:
I suspect the test is wrong here, as `..\n` also contains `.\n`, so the test may pass even without the fix.

Reply:
I changed to `startsWith` which should be fine with the current test. If you think a regex would make it more future proof, let me know.